lib/davinci_pas_test_kit/docs/client_suite_description_v201.md

```diff
@@ -0,0 +1,167 @@
+Thus,
+
+- In order to demonstrate specific workflows, namely pend and denial, users will need to provide the expected
+response for Inferno to send back
+- The current tests do not return to the client all PAS profiles and must support elements to confirm support.
+
+For further details on limitations of these tests, see the *Testing Limitations* section below.
+
+All requests and responses will be checked for conformance to the PAS
+IG requirements individually and used in aggregate to determine whether
+required features and functionality are present. HL7® FHIR® resources are
+validated with the Java validator using `tx.fhir.org` as the terminology server.
+
+## Running the Tests
+
+### Quick Start
+
+For Inferno to simulate a server that always returns approved responses, it needs
+only to know the bearer token that the client will send on requests, for which there are two options.
+
+1. If you want to choose your own bearer token, then
+    1. Select the "2. PAS Client Validation" test from the list on the left.
+    2. Click the '*Run All Tests*' button on the right.
+    3. In the "access_token" field, enter the bearer token that will be sent by the client under test (as part
+        of the Authorization header - Bearer: <provided value>).
+    4. Click the '*Submit*' button at the bottom of the dialog.
+2. If you want to use a client_id to obtain an access token, then
+    1. Click the '*Run All Tests*' button on the right.
+    2. Provide the client's registered id "client_id" field of the input (NOTE, Inferno doesn't support the
+        registration API, so this must be obtained from another system or configured manually).
+    3. Click the '*Submit*' button at the bottom of the dialog.
+    4. Make a token request that includes the specified client id to the
+        `<inferno host>/custom/davinci_pas_client_suite_v201/mock_auth/token` endpoint to get
+        an access token to use on the request of the requests.
+
+In either case, the tests will continue from that point, requesting the user to
+direct the client to make certain requests to demonstrate PAS client capabilities.
+
+Note: authentication options for these tests have not been finalized and are subject to change.
+
+### Complete Setup
+
+The *Quick Start* approach does not test pended or deny workflows. To test these, provide a
+json-encoded FHIR bundle in the "Claim pended response JSON" and "Claim deny response JSON" input field after
+clicking the '*Run All Tests*' button. These responses will be echoed back when a request
+is made during the corresponding test.
+
+### Postman-based Demo
+
+If you do not have a PAS client but would like to try the tests out, you can use
+[this postman collection](https://raw.githubusercontent.com/inferno-framework/davinci-pas-test-kit/blob/main/config/PAS%20Test%20Kit%20Client%20Test%20Demo.postman_collection.json)
```

Comment:
This absolute link doesn't work.  The link in the suite description in the running version on inferno.healthit.gov is currently broken as well.  I think you need to remove the `blob` segment?

Reply:
https://github.com/inferno-framework/davinci-pas-test-kit/blob/main/config/PAS%20Test%20Kit%20Client%20Test%20Demo.postman_collection.json is the link when I navigate to the file, so using that.